examples/src/bin/hmac.rs

```diff
@@ -94,18 +94,18 @@ fn main() -> ! {
         let mut remaining = nsrc;
         hw_hmac.init();
         block!(hw_hmac.configure(HmacPurpose::ToUser, KeyId::Key0)).expect("Key purpose mismatch");
-        let pre_hw_hmac = SystemTimer::now();
+        let pre_hw_hmac = SystemTimer::now(Default::default());
```

Comment:
Not a huge fan of this honestly, this is pretty perplexing without looking into the driver

Reply:
I wasn't too sure either, would you prefer 

- now(Unit::Unit0)
- now() as it was with a new method for accessing unit1's counter

Reply:
I personally would like just `now()`  better - maybe we could have another function which takes a Unit?

Reply:
I'd rename this to `current_timestamp`, keep the unit. We already have `esp_hal::time::now()` and all other nows are rather confuing IMO.

Reply:
I've gone with `SystemTimer::unit_count(Unit)`, I plan on removing all the places in examples where we use `SystemTimer::now` for timing